Description: Fix spectracom log prefixes
 The spectracom driver was logging with the old "wwvb" prefix.
Forwarded: https://gitlab.com/NTPsec/ntpsec/merge_requests/572
Origin: vendor
Author: Richard Laager <rlaager@wiktel.com>
Last-Update: 2017-11-12
--- a/ntpd/refclock_spectracom.c
+++ b/ntpd/refclock_spectracom.c
@@ -286,10 +286,10 @@
 	 */
 	if (temp == 0) {
 		if (up->prev_eol_cr) {
-			DPRINT(2, ("wwvb: <LF> @ %s\n", prettydate(trtmp)));
+			DPRINT(2, ("spectracom: <LF> @ %s\n", prettydate(trtmp)));
 		} else {
 			up->laststamp = trtmp;
-			DPRINT(2, ("wwvb: <CR> @ %s\n", prettydate(trtmp)));
+			DPRINT(2, ("spectracom: <CR> @ %s\n", prettydate(trtmp)));
 		}
 		up->prev_eol_cr = !up->prev_eol_cr;
 		return;
@@ -298,7 +298,7 @@
 	pp->lastrec = up->laststamp;
 	up->laststamp = trtmp;
 	up->prev_eol_cr = true;
-	DPRINT(2, ("wwvb: code @ %s\n"
+	DPRINT(2, ("spectracom: code @ %s\n"
 		   "       using %s minus one char\n",
 		   prettydate(trtmp), prettydate(pp->lastrec)));
 	if (pp->lastrec == 0)
@@ -538,7 +538,7 @@
 #endif /* HAVE_PPSAPI */
 	refclock_receive(peer);
 	record_clock_stats(peer, pp->a_lastcode);
-	DPRINT(1, ("wwvb: timecode %d %s\n", pp->lencode,
+	DPRINT(1, ("spectracom: timecode %d %s\n", pp->lencode,
 		   pp->a_lastcode));
 }
 
